manager/controllers/app/blueprint_controller_test.go

```diff
@@ -59,6 +59,14 @@ var _ = Describe("Blueprint Controller Real Env", func() {
 	Context("Blueprint", func() {
 		BeforeEach(func() {
 			// Add any setup steps that needs to be executed before each test
+			const interval = time.Millisecond * 100
+			blueprint := &app.Blueprint{}
+			Expect(readObjectFromFile("../../testdata/blueprint-read.yaml", blueprint)).ToNot(HaveOccurred())
+			blueprint.SetNamespace("default")
```

Comment:
We don't have tests creating blueprints in the default namespace.

Reply:
I believe this does?  https://github.com/fybrik/fybrik/pull/767/files#diff-bd1268056277f3fcb81d332f4a4294d20e6a8bc3c2e8b0539666ddc5aee13d83R83

Reply:
I could not follow the link...

Reply:
Here's the code snippet from the same file

		It("Test Blueprint Deploy to Correct Namespace", func() {
			deployBlueprint(BlueprintNamespace, true)
		})
		// Blueprints not deployed to fybrik-blueprints should not be successfully reconciled due to the filter preventing
		// reconcile from being called.
		It("Test Blueprint Deploy to Bad Namespace", func() {
			deployBlueprint("default", false)
		})

Reply:
Right, but this test fails to create a blueprint in the default namespace. The purpose of this test is to show that blueprints can only be deployed in a single namespace (fybrik-blueprints). Thus, deleting a blueprint from default namespace is not necessary.

Reply:
Perhaps I'm missing something here, but when I run these tests, and put a watch on blueprints, I see:

```
$ oc get blueprint -A -w
NAMESPACE           NAME                  AGE   READY
fybrik-blueprints   blueprint-with-read   0s    
fybrik-blueprints   blueprint-with-read   0s    
fybrik-blueprints   blueprint-with-read   10s   
fybrik-blueprints   blueprint-with-read   19s   true
fybrik-blueprints   blueprint-with-read   19s   true
default             blueprint-with-read   0s    
default             blueprint-with-read   0s    
fybrik-blueprints   blueprint-with-read   27s   true
```

Reply:
Right, I was confused with another test checking permissions to create objects...